types/src/sector/registered_proof.rs

```diff
@@ -14,6 +14,12 @@ pub enum RegisteredSealProof {
     StackedDRG8MiBV1,
     StackedDRG32GiBV1,
     StackedDRG64GiBV1,
+
+    StackedDRG2KiBV1P1,
```

Comment:
Any clues as to what the P1 at the end stands for? Just curious

Reply:
I made that on our end, they have V1_1 but I didn't want to add underscores in the variant since it's against rust's conventions so I added P1 as in the (p)roofs version 1. Open to suggestions or if you think this should just be `_1` to match exactly and ignore the lint

Reply:
Its cool i was just curious 😀